lib/handlers_v5.js

```diff
@@ -38,10 +38,13 @@ module.exports = {
     complete_request: complete_request,
     inspect_request: inspect_request,
     shutdown_request: shutdown_request,
+    is_complete_request: is_complete_request,
 };
 
 var DEBUG = global.DEBUG || false;
 
+var esprima = require('esprima');
```

Comment:
I was planning to use `acorn` for parsing, but I'm happy to start with `esprima`.

Do you know what are the pros&cons of using one or the other?